client/components/foldable-card/index.jsx

```diff
@@ -96,6 +96,10 @@ var FoldableCard = React.createClass( {
 	},
 
 	renderContent: function() {
+		if ( ! this.state.expanded ) {
```

Comment:
Either of those would be fine for me in this case when it is clearly a boolean check. Regardless, both would improve readability by being in the main render declaration :)